examples/src/keyrings/file_streaming_example.py

```diff
@@ -0,0 +1,155 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""
+This example demonstrates file streaming for encryption and decryption.
+
+File streaming is useful when the plaintext or ciphertext file/data is too large to load into
+memory. Therefore, the AWS Encryption SDK allows users to stream the data, instead of loading it
+all at once in memory. In this example, we demonstrate file streaming for encryption and decryption
+using a Raw AES keyring. However, you can use any keyring for encryption/decryption with streaming.
+
+For more information on how to use Raw AES keyrings, see
+https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/use-raw-aes-keyring.html
+To look at a Raw AES keyring example, checkout out raw_aes_keyring_example.py
```

Comment:
```suggestion

See raw_aes_keyring_example.py in this directory for another raw AES keyring example
in the AWS Encryption SDK for Python.
```

More of a call to action